src/hacbs/components/ApplicationDetails/HacbsApplicationDetails.tsx

```diff
@@ -101,24 +122,56 @@ const HacbsApplicationDetails: React.FC<HacbsApplicationDetailsProps> = ({ appli
                 if (submitClicked) navigate('/app-studio');
               }),
           },
+          {
+            type: 'section-label',
+            key: 'help',
+            label: 'Help',
+          },
           {
             type: 'separator',
             key: 'separator',
             label: '',
           },
           {
             key: 'application-quickstart',
-            label: 'Getting started with an app',
+            label: 'Getting started with an application',
             onClick: () => {
               quickStarts.set('hac-dev', [applicationQuickstartContent]);
               quickStarts.toggle('hacbs-getting-started-app');
             },
           },
           {
-            key: 'learning-resources',
-            label: 'Learning resources',
-            href: '',
-            isDisabled: true,
+            key: 'setup-lifecycle',
+            label: 'Setup application lifecycle',
+            component: (
+              <Link to={`/app-studio/applications/${applicationName}?activeTab=commits`}>
+                Setup application lifecycle
+              </Link>
```

Comment:
I don't understand the logic of navigating to the commits page with the title `Setup application lifecycle`

Reply:
I just kept it as a link as there is nothing defined for this action in the UX doc cc: @Ranelim Please suggest

Reply:
Until we have a use for this action, I suggest we simply comment it out for now.

Reply:
removed